test/query_test.exs

```diff
@@ -29,16 +29,10 @@ defmodule QueryTest do
 
       array = [1, 2, 3]
       assert [[^array]] = query("SELECT array_construct(1, 2, 3)", [])
-
-      #    credo:disable-for-next-line Credo.Check.Design.TagTODO
-      #    TODO: Get working test for OBJECT type
-      #    object = %{a: 1, b: 'BBBB'}
-      #    assert [[^object]] =
-      #             query("SELECT object_construct('a',1,'b','BBBB') as obj", [])
     end
 
     test "long number param", context do
-      assert [["123456789012345678901"]] = query("SELECT ?", [123_456_789_012_345_678_901])
+      assert [[123_456_789_012_345_678_901]] = query("SELECT ?", [123_456_789_012_345_678_901])
```

Comment:
Fixed large number parsing to return an Integer instead of String.

Reply:
@sarahdasko this is probably a breaking change for us.

Reply:
possibly, we'll have to run regression tests on all of our apps to ensure either way.